core/src/i18n/en.yaml

```
@@ -514,12 +514,23 @@ dnsentries:
     ttl: TTL
     dns-name: DNS Name
     text: Text
+    target: Target
     targets: Targets
+    use-cname: Use CNAME
+  name_singular: DNS Entry
   placeholders:
-    targets: Target records (CNAME or A records), one per line
-    text: Text records, one per line
     ttl: Enter the time to live.
-    dns-name: Enter DNS Name
+    dns-name: Enter the DNS Name.
+    target-a: Enter the A record Target.
```

why is target capitalized?

```suggestion
    target-a: Enter the A record target.
```